README.rst

```diff
@@ -1,203 +1,128 @@
 
-+-------------------+----------------+
-| Open edX Release  |  Version       |
-+===================+================+
-|       Juniper     |   <= 1.0       |
-+-------------------+----------------+
-|        Koa        |    > 1.0       |
-+-------------------+----------------+
-|       Lilac       |    > 1.0 < 2.0 |
-+-------------------+----------------+
-|       Maple       |   >= 2.0 < 3.0 |
-+-------------------+----------------+
-|       Nutmeg      |   >= 3.0       |
-+-------------------+----------------+
-|       Olive       |   >= 4.0       |
-+-------------------+----------------+
-|       Palm        |   >= 5.0       |
-+-------------------+----------------+
-|       Quince      |   >= 6.0       |
-+-------------------+----------------+
+- **Post to Webhook URL:** Sends data to a specified URL based on the eox-hooks settings.
+- **Get Extended Certificate Data:** Adds comprehensive certificate information to the webhook request.
+- **Trigger Enrollments Creation:** Initiates an asynchronous task to enroll users in a list of courses.
+- **Trigger Grades Assignment:** Propagates grades to a specified course component.
+- **Create Enrollments for Program:** Enroll users in a predefined list of courses constituting a Course Program.
 
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
 
+Installation
+=============
 
-**Juniper**
+#. Add this plugin in your Tutor ``config.yml`` with the ``OPENEDX_EXTRA_PIP_REQUIREMENTS`` setting.
 
-.. code-block:: yaml
-
-   EOX_HOOKS_ENROLLMENTS_BACKEND: "eox_hooks.edxapp_wrapper.backends.enrollments_l_v1"
-   EOX_HOOKS_COURSES_BACKEND: "eox_hooks.edxapp_wrapper.backends.courses_l_v1"
-   EOX_HOOKS_COURSE_MODES_BACKEND: "eox_hooks.edxapp_wrapper.backends.course_modes_l_v1"
-   EOX_HOOKS_MODELS_BACKEND: "eox_hooks.edxapp_wrapper.backends.models_l_v1"
-
-**Koa**
-
-.. code-block:: yaml
-
-   EOX_HOOKS_ENROLLMENTS_BACKEND: "eox_hooks.edxapp_wrapper.backends.enrollments_l_v1"
-   EOX_HOOKS_COURSES_BACKEND: "eox_hooks.edxapp_wrapper.backends.courses_l_v1"
-   EOX_HOOKS_COURSE_MODES_BACKEND: "eox_hooks.edxapp_wrapper.backends.course_modes_l_v1"
-   EOX_HOOKS_MODELS_BACKEND: "eox_hooks.edxapp_wrapper.backends.models_l_v1"
-
-**Lilac**
-
-.. code-block:: yaml
-
-   EOX_HOOKS_ENROLLMENTS_BACKEND: "eox_hooks.edxapp_wrapper.backends.enrollments_l_v1"
-   EOX_HOOKS_COURSES_BACKEND: "eox_hooks.edxapp_wrapper.backends.courses_l_v1"
-   EOX_HOOKS_COURSE_MODES_BACKEND: "eox_hooks.edxapp_wrapper.backends.course_modes_l_v1"
-   EOX_HOOKS_MODELS_BACKEND: "eox_hooks.edxapp_wrapper.backends.models_l_v1"
+   .. code-block:: yaml
+      
+      OPENEDX_EXTRA_PIP_REQUIREMENTS:
+         - eox-hooks=={{version}}
+         
+#. Save the configuration with ``tutor config save``.
+#. Build the image and launch your platform with ``tutor local launch``.
```

Comment:
```suggestion
#. Build the image with ``tutor images build openedx --no-cache``
#. Launch your platform with ``tutor local launch``.
```

Reply:
@BryanttV: why is the `--no-cache` flag needed?

Reply:
We wouldn't need the command to build the image since `tutor dev launch` already does it, right? :thinking:

Reply:
Good point, @BryanttV, yes. With the tutor launch, we run the build process.

I'll resolve this conversation, but feel free to reopen it if you need to add more comments.

Ref: https://docs.tutor.edly.io/reference/cli/local.html#tutor-local-launch